src/rest-server/src/models/kubernetes/kubernetes.js

```diff
@@ -95,15 +96,12 @@ const getNodes = async () => {
 };
 
 const getPods = async (options = {}) => {
-  const {labelSelector, negativeLabelSelector, namespace} = options;
+  const {namespace} = options;
   const client = getClient();
-  const labelSelectorStr = encodeSelector(labelSelector, negativeLabelSelector);
+
   const requestOptions = {};
-  if (labelSelectorStr) {
-    requestOptions.params = {
-      labelSelector: labelSelectorStr,
-    };
-  }
```

Comment:
seems you can use `options` object directly if you do not need to generate `labelSelectorStr` anymore